core-bundle/src/Command/MigrateCommand.php

```diff
@@ -362,6 +364,19 @@ private function executeMigrations(bool &$dryRun, bool $asJson, string $specifie
                 // Do not run the update recursive if a hash was specified
                 break;
             }
+
+            if ($loopControl-- < 1) {
+                if ($asJson) {
+                    $this->writeNdjson('error', [
+                        'message' => 'The migrations were stopped after 19 iterations as a precaution. There is a high chance of an infinite loop of migrations.',
+                        'isSuccessful' => false,
+                    ]);
+                } else {
+                    $this->io->error('The migrations were stopped after 19 iterations as a precaution. There is a high chance of an infinite loop of migrations. If this is not the case, please re-run the command. To troubleshoot this error, check the `shouldRun()` method of the migration(s) listed above.');
```

Comment:
```suggestion
                    $this->io->error('The migrations were stopped after 19 iterations as a precaution. There is a high chance of an infinite loop of migrations. If this is not the case, please re-run the command. To troubleshoot this error, check the shouldRun() method of the migration(s) listed above.');
```

Does `SymfonyStyle` support backticks? Otherwise we should probably use double quotes or an option like `<options=bold>`.